eastlake/steps/delete_images.py

```diff
@@ -65,13 +66,16 @@ def execute(self, stash, new_params=None):
                         if (seg_file is not None):
                             if os.path.isfile(seg_file):
                                 self.logger.error("removing file %s" % seg_file)
+                                os.remove(seg_file)
```

Comment:
This change is also more of a BUG than an ENH—there was a logging statement about removing a file but no call to actually remove that file.

Reply:
Arg woops.